### PR TITLE
{bio}[intel/2017a] EMBOSS 6.6.0 w/ X11-20170314 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
ca043.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c293c5aa27cfd39fd60aed6c950dabd7 for a full test report.

File: easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ the needs of the molecular biology (e.g. EMBnet) user community."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'ftp://emboss.open-bio.org/pub/EMBOSS/',
+    'ftp://emboss.open-bio.org/pub/EMBOSS/old/%(version_major_minor)s.0',
+]
+
+sources = [SOURCE_TAR_GZ]
```
Comment (verdurin) [inline easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb]: @hajgato add a checksum please...

File: easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+patches = ['EMBOSS_disable-embossupdate.patch']
+
+dependencies = [
+    ('X11', x11ver),
+    ('libgd', '2.2.4'),
+    ('libharu', '2.3.0'),
+    ('Java', '1.8.0_121', '', True),
```
Comment (verdurin) [inline easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb]: @hajgato consider using the most up to date `Java`?

File: easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb]: Using EasyBuild v3.4.0: `eb EMBOSS-6.6.0-intel-2017a-X11-20170314.eb --inject-checksums`

Comment (hajgato) [reply easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb]: @verdurin Done!

File: easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb (hunk 2 above)
Comment (hajgato) [reply easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb]: @verdurin Done!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca022.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0e5cc52b4865a2f6d0001625644c3b69 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @hajgato!

**Pull request merged**